docs/cunumeric/source/conf.py

```diff
@@ -100,6 +100,12 @@
     ("py:class", "N"),
     ("py:class", "nested list of array_like"),
     ("py:class", "scalars"),
+    ("py:class", "complex ndarray"),
+    ("py:class", "sequence of ints"),
+    ("py:class", "backward"),
```

Comment:
do we need these (backward, ortho, and forward) in the blocklist despite the quotes in fft.py?

Reply:
I think it would be safe to have them here in case documentation for the future ffit routines wouldn't have quotes for "backward" etc.

Reply:
Well I don't think it makes anything safe, but it rather makes those unquote keywords uncaught by the build. Note that these keywords are not the names of types. They are string literals that have special meaning. So any docstring that refers to them needs to quote them.

Reply:
FWIW I would have preferred to backquote the individual values, rather than the entire `{...}` but I quoted everything to placate Sphinx without having to add a nitpick exception. I do think Sphinx has gotten too aggressive with auto linking in the last few years, but that's just the way things are at present.

Reply:
So should I comment individual values as well?

Reply:
I don't think so (I think it would confuse sphinx), I was just trying to explain the rationale for the current state of things